apiserver/common/crossmodel/state.go

```diff
@@ -23,6 +23,9 @@ type statePoolShim struct {
 
 func (p *statePoolShim) Get(modelUUID string) (Backend, func(), error) {
 	st, releaser, err := p.StatePool.Get(modelUUID)
+	if err != nil {
+		return stateShim{}, func() {}, err
```

Comment:
I think this can just be
return nil, func() {}, err

Reply:
True :) I have copied whatever was done in other error returns. 
Changed to nil now.